xbmc/video/VideoInfoTag.cpp

```diff
@@ -549,7 +550,7 @@ void CVideoInfoTag::Serialize(CVariant& value) const
   value["playcount"] = m_playCount;
   value["lastplayed"] = m_lastPlayed.IsValid() ? m_lastPlayed.GetAsDBDateTime() : StringUtils::Empty;
   value["top250"] = m_iTop250;
-  value["year"] = m_iYear;
+  value["year"] = m_premiered.GetYear();
```

Comment:
What does `CDateTime::GetYear()` return if the `CDateTime` object is invalid? It must be `0` or it needs to be properly handled here.